app/scripts/lib/transaction/swaps.ts

```diff
@@ -0,0 +1,122 @@
+import { pickBy, merge } from 'lodash';
+import {
+  TransactionStatus,
+  TransactionType,
+  TransactionMeta,
+} from '../../../../shared/constants/transaction';
+import MetamaskController from '../../metamask-controller';
+
+type SwapOptions = {
```

Comment:
As this type is constructed externally, do we need to `export` it and add JSDoc?

Reply:
Fixed in https://github.com/MetaMask/metamask-extension/pull/21456/commits/86dfac2fc9acf571f3240e2ea59948f8ed6b4a1f